feature-libs/order/components/order-details/order-overview/order-overview.component.html

```diff
@@ -81,20 +81,6 @@
         </div>
       </ng-container>
 
-      <div class="cx-summary-card">
```

Comment:
don't remove them. This component is also used in b2b.

Reply:
Done